tfquaternion/tfquaternion.py

```diff
@@ -386,7 +386,7 @@ def validate_type(x):
     @scope_wrapper
     def norm(self, keepdims=True):
         """Return the norm of the quaternion."""
-        return tf.reduce_sum(tf.square(self._q), axis=-1, keep_dims=keepdims)
+        return tf.reduce_sum(tf.square(self._q), axis=-1, keepdims=keepdims)
```

Comment:
This breaks the code for older tf versions (I think somewhere around 1.4). Could you maybe add an if statement, something along the lines `if tf.__version__.split('.')[1] < ` to use `keep_dims/keepdims` approrietly?

Reply:
If you want to have backwards compatibility I would advice just to stick with keep_dims for now, it is still an alias to avoid such conditionals. I personally just got annoyed by the deprecation warning at TensorFlow startup. The other half of my pull request is the more important one anyways, probably should have opened two requests instead of one.

Reply:
Sorry for the late reply! Yeah, I also got annoyed, which is why I considered such a hack. Apparently tfquaternion doesn't work with older versions of tensorflow anyway (see #3), so I might just merge this in. I'll might have more time tomorrow to check ;)

Reply:
I am actually using it right now on a cluster on which I can only run TF 1.4, which still needs keep_dims, and except for that everything works the same. Which is kind of funny because now I need keep_dims myself.